embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/event/EmbraceEventService.kt

```diff
@@ -268,13 +263,13 @@ internal class EmbraceEventService(
 
     override fun close() {
         cleanCollections()
-        logDeveloper("EmbraceEventService", "close")
     }
 
     override fun cleanCollections() {
-        eventIds.clear()
-        activeEvents.clear()
-        logDeveloper("EmbraceEventService", "collections cleaned")
+        val activeEventIds = activeEvents.values.map { it.event.eventId }
```

Comment:
Does the active moments list get updated on timeout?

Reply:
Yep, the `EventDescription` contains a callback to `endEvent` that is invoked on the timeout. The item is removed from the collection at that point